1

```diff
@@ -0,0 +1,33 @@
+pick 4bce78b29c bugfix: make bitmap to vairiant more understandable
```

Comment:
Looks like it was commited accidentally, please delete this file

Reply:
Done. Sorry for that.